neuropoly/axondeepseg
# Fix Naming Convention

Fixes #439 


**OSF: -** Test files to upload on OSF 
[__test_files__.zip](https://github.com/neuropoly/axondeepseg/files/5849453/__test_files__.zip)

**DONE:**
- [x] fix naming convention on FSLeyes plugin
- [x] fix naming convention in Notebooks
- [x] Fix naming convention in [apply_model.py](https://github.com/neuropoly/axondeepseg/blob/master/AxonDeepSeg/apply_model.py) scipt
- [x] Upload test files on OSF

**TODO:**
- [ ] Add documentation on Wiki



 **To test this PR :**
 
  The segmented images names should follow a common convention, that is
 1. **image_name_seg-axonmyelin.png** (axon +myelin segmented mask)
 2. **image_name_seg-axon.png** (axon mask)
 3.  **image_name_seg-myelin.png** (myelin mask)

To check if the naming conventions are being followed, for the cases given below ADS should segment images adhering to the naming convention.

 1.  **FSLeyes**: Test `Apply ADS Segmentation Model` and `Save segmentation` buttons and check if they are following the naming convention.
 2. **Notebooks** : Run all the notebooks and check whether the naming conventions are being followed.
 3.  **CLI**: This has been tested in the ADS unit tests, so you should expect all the test cases to pass.
